touchables/TouchableNativeFeedback.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+/**
+ * This implementation is a copy of react-native behavior.
```

Comment:
In this case can we import `TouchableNativeFeedback` directly from RN and export it here instead of copying the code?